src/libutil/experimental-features.cc

```diff
@@ -254,6 +254,13 @@ constexpr std::array<ExperimentalFeatureDetails, numXpFeatures> xpFeatureDetails
             Allow the use of the `read-only` parameter in [local store](@docroot@/store/types/local-store.md) URIs.
         )",
     },
+    {
+        .tag = Xp::LocalOverlayStore,
+        .name = "local-overlay-store",
+        .description = R"(
+            Allow the use of [local overlay store](@docroot@/command-ref/new-cli/nix3-help-stores.md#local-overlay-store).
+        )",
```

Comment:
```suggestion
        )",
        .trackingUrl = ""https://github.com/NixOS/nix/milestone/50",
```

(to add once this is rebased on top of `master`)